api/spec/controllers/users.spec.js

```diff
@@ -8,18 +8,18 @@ describe("/users", () => {
     await User.deleteMany({});
   });
 
-  describe("POST, when email and password are provided", () => {
+  describe("POST, when email, password and username are provided", () => {
     test("the response code is 201", async () => {
       let response = await request(app)
         .post("/users")
-        .send({email: "poppy@email.com", password: "1234"})
+        .send({email: "poppy@email.com", password: "1234", username: 'person1'})
       expect(response.statusCode).toBe(201)
     })
 
     test("a user is created", async () => {
       await request(app)
         .post("/users")
-        .send({email: "scarlett@email.com", password: "1234"})
+        .send({email: "scarlett@email.com", password: "1234", username: 'person1'})
       let users = await User.find()
       let newUser = users[users.length - 1]
       expect(newUser.email).toEqual("scarlett@email.com")
```

Comment:
What happens we try to create a user with the username missing? If we want to get some kind of validation error, similar to what happens when there's a missing email or password, we could add one additional test below, similar to "POST, when password is missing", but with the username, and assert we get a client error 400 — but I'm picky here :)